model/dao/TemplateDAO/PostgreSQLTemplateDAO.php

```diff
@@ -0,0 +1,237 @@
+<?php
+/*
+ * Copyright (C) 2009-2013 Igalia, S.L. <info@igalia.com>
+ *
+ * This file is part of PhpReport.
+ *
+ * PhpReport is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PhpReport is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+/** File for PostgreSQLTaskDAO
+ *
+ *  This file just contains {@link PostgreSQLTaskDAO}.
+ *
+ * @filesource
+ * @package PhpReport
+ * @subpackage DAO
+ * @author Jorge López Fernández <jlopez@igalia.com>
+ * @author Jacobo Aragunde Pérez <jaragunde@igalia.com>
```

Comment:
We are not even using these `@author` tags lately, git information about authorship is more accurate (and gets updated for free!). Please, remove them :)